plugin/storage/es/spanstore/reader_test.go

```diff
@@ -144,35 +144,35 @@ func TestSpanReaderIndices(t *testing.T) {
 	date := time.Now()
 	dateFormat := date.UTC().Format("2006-01-02")
 	testCases := []struct {
-		indices []string
+		index string
 		params  SpanReaderParams
 	}{
 		{params:SpanReaderParams{Client:client, Logger: logger, MetricsFactory: metricsFactory,
 			IndexPrefix:"", Archive: false},
-			indices: []string{spanIndex+dateFormat}},
+			index: spanIndex+dateFormat},
 		{params:SpanReaderParams{Client:client, Logger: logger, MetricsFactory: metricsFactory,
 			IndexPrefix:"", UseReadWriteAliases: true},
-			indices: []string{spanIndex+"read"}},
+			index: spanIndex+"read"},
 		{params:SpanReaderParams{Client:client, Logger: logger, MetricsFactory: metricsFactory,
 			IndexPrefix:"foo:", Archive: false},
```

Comment:
nit: Change the IndexPrefix values so just "foo" rather than including ':'.

Reply:
It's there to stress that `:` would be added if it is specified

Reply:
ok